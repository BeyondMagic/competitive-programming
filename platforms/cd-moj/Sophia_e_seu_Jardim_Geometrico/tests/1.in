5 12
AYooouFoooaH
GoDauooGAJoK
kuoHuABdCoNG
AA5ooOAooKPd
l-DYOoo3MudF
27
t A
t a
r 5 12
r 3 7
t D
r 4 3
p o 1 6
p o 1 7
t k
t -
r 1 7
r 1 6
t 5
r 5 1
t G
t H
t P
t K
r 5 8
r 5 9
t u
t Y
t O
t d
r 2 10
t C
t N
