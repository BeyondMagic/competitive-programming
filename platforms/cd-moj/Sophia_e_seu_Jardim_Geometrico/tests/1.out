44
..ooo..ooo..
.o...oo...o.
..o......o..
...oo..oo...
.....oo.....
